influxql/functions.go

```diff
@@ -355,11 +347,19 @@ type FloatHoltWintersReducer struct {
 }
 
 const (
-	defaultAlpha   = 0.5
-	defaultBeta    = 0.5
-	defaultGamma   = 0.5
-	defaultPhi     = 0.5
+	// Arbitrary weight for initializing some intial guesses.
+	// This should be in the  range [0,1]
+	weight = 0.5
```

Comment:
Is there a way these variables can be prefixed for with something like `hw` to namespace the holt winters variables from other variables?